balance/controllers/address.js

```diff
@@ -41,7 +41,7 @@ module.exports.configuration = {
 
 v2.walletBalance =
 { handler: (runtime) => {
-  return async (request, reply) => {
+  return async (request, h) => {
```

Comment:
in all these functions where `h` is unused, is there any particular reason to include it at all?

Reply:
consistency with the expected handler function signature